lib/Image/PNG/Portable.rakumod

```diff
@@ -34,12 +33,12 @@ method set(
 }
 
 method set-all(Int:D $r, Int:D $g, Int:D $b, Int:D $a = 255) {
+    self!init-if-needed;
     my $buffer = $!data;
     my $index = 0;
     my $alpha = $!alpha;
 
     for ^$!height {
-        # every line offset by 1 again for filter byte
```

Comment:
Why remove these comments?  Are they incorrect or unclear?

Reply:
I removed this when I considered implementing filters on write but forgot to put it back. Fixed.